src/less/border.less

```diff
@@ -0,0 +1,6 @@
+@borderWidthSmall: .0625rem; // 1px
+@borderWidthMedium: .125rem; // 2px
+@borderWidthLarge: .1875rem; // 3px
+
+@borderRadiusSmall: .125rem; // 2px
+@borderRadiusMedium: .1875rem; // 3px
```

Comment:
I wonder if we should be more explicit about these similar to how we do the spacing file. Thoughts?